dev/protos/tests/interoperability/src/proto/grpc/testing/empty.proto

```diff
@@ -1,5 +1,4 @@
-
-// Copyright 2015 gRPC authors.
+// Copyright 2015, gRPC Authors All rights reserved.
```

Comment:
The proto files comes from upstream repos (some can be updated via a script) so we shouldn't modify their copyright headers.

Reply:
I've backed out the changes and added the any `.proto` file under `./dev/protos/` to the ignore list.